frontend/src/static/js/main.js

```diff
@@ -136,7 +144,7 @@ window.app = new window.Vue({
       if (Object.keys(info).length === tabInfoLength) {
         this.$store.commit('updateTabAuthorshipInfo', info);
       } else if (hash.tabOpen === 'false' || tabInfoLength > 2) {
-        window.app.isTabActive = false;
```

Comment:
Recently, we changed `isTabActive` to a computed value from Vuex, can you do a check if the `isTabActive` here is affected or not?

Reply:
Which PR was it?

Reply:
https://github.com/reposense/RepoSense/pull/1480/files Can you check this PR, it is good to fix it once and for all.

Reply:
Made some changes to how it works now.